dysnix/bsc/values.yaml

```diff
@@ -0,0 +1,140 @@
+# Default values for bsc.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+terminationGracePeriodSeconds: 180
+
+image:
+  repository: dysnix/bsc
+  tag: latest
+  pullPolicy: Always
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+# enable this only once - when your blockchain is empty. It overwrites everything
+init_genesis: true
+
+# Don't use spaces or special chars
+bsc:
+  # we don't have other chains, it's just a pod label for now
+  chain: testnet
+  #  Blockchain sync mode ("fast", "full", or "light") (default: fast)
+  syncmode: "full"
+  # Blockchain garbage collection mode ("full", "archive") (default: "full")
+  gcmode: "full"
+  base_path: "/data"
+  rpcApi: "eth,net,web3,txpool,parlia"
+  rpcVhosts: "*"
+  wsApi: "net,web3,eth"
+  wsOrigins: "*"
+  graphQlVhosts: "*"
+  maxpeers: 50
+  cache: 8192
+
+livenessProbe:
+  initialDelaySeconds: 300
+  periodSeconds: 300
+  timeoutSeconds: 10
+  successThreshold: 1
+  failureThreshold: 2
+
+# readinessProbe:
+#   initialDelaySeconds: 300
+#   periodSeconds: 300
+#   timeoutSeconds: 10
+#   successThreshold: 1
+#   failureThreshold: 2
+
+service:
+  type: ClusterIP
+  rpcPortName: &rpcPortName jsonrpc
+  rpcPort: &rpcPort 8575
+  wsPort: &wsPort 8576
+  wsPortName: &wsPortName web-socket
+  graphQlPort: &graphQlPort 8577
+  graphQlPortName: &graphQlPortName qraphql
+  p2pPort0: &p2pPort0 30311
+  p2pPortName0: &p2pPortName0 p2p
+  p2pPortProtocol0: &p2pPortProtocol0 TCP
+  p2pPort1: &p2pPort1 30311
+  p2pPortName1: &p2pPortName1 p2p-discovery
+  p2pPortProtocol1: &p2pPortProtocol1 UDP
+  ports:
+    - port: *rpcPort
+      name: *rpcPortName
+    - port: *wsPort
+      name: *wsPortName
+    - port: *graphQlPort
+      name: *graphQlPortName
+    - port: *p2pPort0
+      name: *p2pPortName0
+      protocol: *p2pPortProtocol0
+    - port: *p2pPort1
+      name: *p2pPortName1
+      protocol: *p2pPortProtocol1
+
+bscCmdOverride: false
+bscCmd: ""
+
+externalLB: false
+externalLBIP: ""
+externalLBSourceRanges: {}
+
+externalLBp2p: false
+externalLBp2pIP: ""
+
+externalLBp2pDiscovery: false
+externalLBp2pDiscoveryIP: ""
+
+internalLB: false
+internalLBIP: ""
+
+persistence:
+  enabled: false
```

Comment:
Why do we use persistence disabled by default?

Reply:
Good catch. 👍  Fixed.